mbuild/formats/cassandramcf.py

```diff
@@ -91,11 +91,11 @@ def write_mcf(structure, filename, angle_style,
         if len(structure.adjusts) > 0:
             coul14 = structure.adjusts[0].type.chgscale
         else:
-            coul14 = 1.0
+            coul14 = 0.0
             if (len(structure.dihedrals) > 0 or
                     len(structure.rb_torsions) > 0):
                 warnings.warn('Unable to infer coulombic 1-4 '
-                        'scaling factor. Setting to 1.0')
+                        'scaling factor. Setting to 0.0')
```

Comment:
It is probably a better practice to use these `coul14` and `lj14` variables in these warnings so we have less code to edit in the future.

```suggestion
                        'scaling factor. Setting to {0:.1f}'.format(coul14))
```